openmdao/docs/features/building_blocks/components/spline_comp.rst

```diff
@@ -132,24 +132,40 @@ specify the number of control points using the 'num_cp' argument.
 
 .. embed-code::
     openmdao.components.tests.test_spline_comp.SplineCompFeatureTestCase.test_bspline_options
-    :layout: code
+    :layout: code, output
 
 
 SplineComp Interpolation Distribution
 -------------------------------------
 
-We have included three different distribution functions for users to replicate functionality that used to
-be built-in to the individual akima and bsplines components. The `cell_centered` function takes the number
-of cells, and the start and end values, and returns a vector of points that lie at the center of those
-cells. The 'node_centered' function reproduces the functionality of numpy's linspace.  Finally, the
-`sine_distribution` function creates a sinusoidal distribution, in which points are clustered towards the
-ends. A 'phase' argument is also included, and a phase of pi/2.0 clusters the points in the center with
-fewer points on the ends.
+The `cell_centered` function takes the number of cells, and the start and end values, and returns a
+vector of points that lie at the center of those cells. The 'node_centered' function reproduces the
+functionality of numpy's linspace.  Finally, the `sine_distribution` function creates a sinusoidal
+distribution, in which points are clustered towards the ends. A 'phase' argument is also included,
+and a phase of pi/2.0 clusters the points in the center with fewer points on the ends.
+
+.. note::
```

Comment:
I would move the note up above the new paragraph (though again, verify with justin)

Reply:
He's fine with this